.lando.yml

```
@@ -4,9 +4,21 @@ config:
   php: '7.4'
   via: apache:2.4
   webroot: web
-  database: mariadb:10.4
   xdebug: false
```

```suggestion
  database: mariadb:10.4
  xdebug: false
```
In my testing I  found that this `database` line doesn't actually need to be deleted.